build.gradle

```diff
@@ -42,7 +42,7 @@ project.ext {
     minSdkVersion = 10
     targetSdkVersion = 23
 
-    supportVersion = "23.2.1"
+    supportVersion = "24.0.0"
```

Comment:
"Support Library should always match the compileSdkVersion even if the targetSdkVersion or minSdkVersion are lower." [http://stackoverflow.com/a/32939791/5369600]